test/e2e-go/cli/goal/expect/rekeyTest.exp

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/expect -f
+#exp_internal 1
+set err 0
+log_user 1
+
+source goalExpectCommon.exp
+source algokeyExpectCommon.exp
+
+set TEST_ALGO_DIR [lindex $argv 0]
+set TEST_DATA_DIR [lindex $argv 1]
+
+proc rekeyTest { TEST_ALGO_DIR TEST_DATA_DIR } {
+    set TIME_STAMP [clock seconds]
+
+    set TEST_ROOT_DIR $TEST_ALGO_DIR/root_$TIME_STAMP
+    set TEST_PRIMARY_NODE_DIR $TEST_ROOT_DIR/Primary/
+    set NETWORK_NAME test_net_expect_$TIME_STAMP
+    set NETWORK_TEMPLATE "$TEST_DATA_DIR/nettemplates/TwoNodes50EachFuture.json"
+    set PRIMARY_WALLET_NAME unencrypted-default-wallet
+    set PRIMARY_WALLET_PASSWORD ""
+    set TRANSFER_AMOUNT 100000
+    set FEE_AMOUNT 1000
+    set TXN_OUTFILE /tmp/ntxn
+    set SIGNED_TXN_OUTFILE /tmp/nstxn
```

Comment:
TODO:  outfiles should be in a more dynamic location such as those created by `mktemp`.

Reply:
Fixed